common/app/views/fragments/collections/popular.scala.html

```diff
@@ -51,7 +51,8 @@
         }
 
         @if(showMPU(containerDefinition)) {
-            <div class="fc-slice__popular-mpu js-fc-slice-mpu-candidate fc-slice__item--mpu-candidate fc-slice__item--no-mpu"></div>
+            <div class="fc-slice__popular-mpu js-fc-slice-mpu-candidate fc-slice__item--mpu-candidate fc-slice__item--mpu-candidate--desktop">
```

Comment:
It seems that `js-fc-slice-mpu-candidate` is also responsible for adding the MPU into 'most viewed' when it appears at the bottom of articles... :cry:

I can simplify the other classes: 
```
<div class="fc-slice__popular-mpu js-fc-slice-mpu-candidate fc-slice__item--mpu-candidate">
```